sources/TileDB.CSharp/Obsoletions.cs

```diff
@@ -29,5 +29,8 @@ internal static class Obsoletions
 
         public const string QueryConditionCombineMessage = "The QueryCondition.Combine method is obsolete. Use the '&', '|' and '!' operators instead.";
         public const string QueryConditionCombineDiagId = "TILEDB0008";
+
+        public const string ConsolidateMetadataMessage = "The Array.ConsolidateMetadata and ArrayMetadata.ConsolidateMetadata methods are obsolete. Do a regular consolidation with the config value 'sm.consolidation.mode' set to 'array_meta' instead.";
```

Comment:
Just a nit, but I think `Prefer Array.Consolidate with the config value 'sm.consolidation.mode' set to 'array_meta' instead.` or something similar would give a more direct resolution to users.

Reply:
Good idea, I replaced "prefer" with "call".